framework/src/base/Assembly.C

```diff
@@ -3921,6 +3934,8 @@ Assembly::setCachedJacobianContributions()
     {
       // First zero the rows (including the diagonals) to prepare for
       // setting the cached values.
+      // If we did not cache anything yet, we should skip
+      // if (_cached_jacobian_contribution_rows[tag].size())
```

Comment:
Are we not skipping? If not, then we can remove this comment and code?

Reply:
I will delete it